### DISCUSSION TITLE
mouse click controls for event's in and out times + shifts

### DISCUSSION THREAD

**Discussion by PreetM07** [SubtitleEdit/subtitleedit#4969] [Q&A] [answered]
Hello

I have found that in the latest version of the SE alt+left click on the waveform moves the event (only the selected event) to that position and not just the start time. While if I am not wrong earlier alt+click was to changing the start time only (that was too helpful.)

Now settings > shortcuts doesn't show or allows to edit this mouse click behaviors on the waveform (Please pardon me if there is, would be much helpful if anyone can let me know where it is.) I wanted to check if there is any other method to customize this click behaviors? (maybe a feature with two shortcut keys?)

PS:
ctrl + left click = set end time of active event
alt + left click = move the active event start time at click position
ctrl + shift + left click = set start and offset the rest
ctrl + alt + left click = does nothing
(any combination) + right click = normal right click

**Comment by OmrSi** [comment, marked as answer]
These settings are embedded in SE and the user doesn't have the option to modify them right now.
You can see a list of them in SE's help page: https://www.nikse.dk/SubtitleEdit/Help#waveform
Which are:
```
Mouse single click+Shift will set start for selected line (not moving end time)
Mouse single click+Ctrl will set end for selected line (not moving start time)
Mouse single click+Alt will set start for selected line (and keep duration)
Mouse double click on non-paragraph - toggle play
Mouse single click+Ctrl+Shift will set of start selected line + offset all lines after selected line (keeping durations)
Holding down ALT when moving start/end will move nearest subtitle start/end too if it's closer than 500 milliseconds - like this.
```

**Reply by PreetM07** [reply]
Oh, so dumb of me!! how did I missed testing shift + click. 
Thanks @OmrSi - again helping me out.

**Reply by OmrSi** [reply]
My pleasure. :D
